### DISCUSSION TITLE
Teacher verification benefits

### DISCUSSION THREAD

**Discussion by jtocch** [community/community#41842] [GitHub Education]
Hi there,
I'm a passionate Digital Technologies educator from Melbourne Australia trying to apply for teacher verification benefits. My application cannot be reviewed as the school I selected doesn’t appear to have a campus location in my country.

If you can assist me in this matter and advise me of my next steps that would be greatly appreciated.

Have also sent a request for this previously and the matter still remains unresolved. As I'm in the middle of curriculum development for coding and game development would love to have access to these resources for ideas and inspiration.

Kind Regards
Jenny

**Comment by samus-aran** [comment]
Hi @jtocch 👋 

We're happy you're here. Often the application process accepts remote faculty as long as you met the school's minimum evidence requirements e.g. academic email or dated school ID. Do you have any screenshots of the error? Let us know how you get on.
